src/Components/VideoMenu/VideoMenu.jsx

```diff
@@ -0,0 +1,35 @@
+import "./videomenu.css";
+import {
+  MdOutlineWatchLater,
+  CgPlayList,
+  CgPlayListCheck,
+  BiLike,
+  AiFillLike,
+  RiHistoryLine,
+} from "../../Utils/icons";
+import { useLocation, useNavigate } from "react-router-dom";
+import { useAuth } from "../../Context/Auth/auth-context";
+import { useVideos } from "../../Context/VideosContext/VideosContext";
```

Comment:
Suggestion: You can make a separate file for exporting context from there

Reply:
Yeah , I'll do that